contribs/drt-extensions/src/main/java/org/matsim/contrib/drt/extension/reconfiguration/logic/DefaultCapacityReconfigurationLogic.java

```diff
@@ -129,7 +129,7 @@ public DefaultCapacityReconfigurationLogic(FleetSpecification fleetSpecification
 				maxServiceEndTime = vehicleSpecification.getServiceEndTime();
 			}
 			for (int slotIndex = timeToSlotIndex(
-					vehicleSpecification.getServiceBeginTime()); slotIndex < timeToSlotIndex(
+					vehicleSpecification.getServiceBeginTime()); slotIndex <= timeToSlotIndex(
```

Comment:
i forgot to mention this bug fix here. Before, the last possible time slot wasn't handled properly